Pipfile

```diff
@@ -6,6 +6,9 @@ name = "pypi"
 [packages]
 gitpython = "*"
 requests = ">=2.20.0"
+pytest = "*"
+"flake8" = "*"
```

Comment:
These are added to an incorrect section of the Pipfile. This ***must*** be fixed -- these are not run-time dependencies, but development dependencies!

Reply:
@Michionlion I have removed these lines of code, I don't really think they were necessary as they already appear in `[dev-packages]`.